docs/source/user-guide/tabular/tutorials/plot_quick_train_test_validation.py

```diff
@@ -0,0 +1,199 @@
+# -*- coding: utf-8 -*-
+"""
+Train Test Validation Scenario on Lending Club Data - 5 min Quickstart
+***********************************************************************
+
+The deepchecks train test validation suite is relevant any time you have data splits
+that you wish to validate:
+whether it's for comparing distributions across different data splits
+(e.g. before training a model or when splitting data for cross-validation),
+or for comparing new data batch to previous data batches.
+Here we'll use data from the lending club dataset, to demonstrate how you can run
+the suite with only a few simple lines of code, and see which kind of insights it can find.
+
+.. code-block:: bash
+
+    # Before we start, if you don't have deepchecks installed yet,
+    # make sure to run:
+    pip install deepchecks -U --quiet #--user
+"""
+
+#%%
+# Load Data and Prepare Data
+# ====================================================
+#
+# Load Data
+# -----------
+
+
+from deepchecks.tabular import datasets
+import pandas as pd
+
+# TODO - load this from datasets...
+# data = datasets.regression.avocado.load_data(data_format='DataFrame', as_train_test=False)
+data = pd.read_csv("lc-2017-18.csv", parse_dates=['issue_d'])
+data.head(2)
+
+
+#%%
+# Define Lending Club Metadata
+# ------------------------------
+
+categorical_features = ['addr_state', 'application_type', 'home_ownership', \
+  'initial_list_status', 'purpose', 'term', 'verification_status', 'sub_grade']
+index_name = 'id'
+label = 'loan_status' # 0 is DEFAULT, 1 is OK
+datetime_name = 'issue_d'
+
+
+#%%
+# Split Data
+# -------------
+
+# Use data for from June and July for train and August for test:
+train_df = data[data['issue_d'].dt.month.isin([6, 7])]
```

Comment:
does it work? pandas recognize this feature as a timestamp automatically?

Reply:
Thatis' why I load the dataframe with parse_dates=['issue_d']
(and yes, it parses correctly. can define format explicitly but given that it's quite standard dropped that)

Reply:
Cool